plugins/event_notification/lib/kEventNotificationFlowManager.php

```diff
@@ -151,15 +151,7 @@ public function shouldConsumeEvent(KalturaEvent $event)
 			
 			$scope->resetDynamicValues();
 			
-			$notificationParameters = $notificationTemplate->getContentParameters();
-			foreach($notificationParameters as $notificationParameter)
-			{
-				/* @var $notificationParameter kEventNotificationParameter */
-				if(!is_null($notificationParameter->getValue()))
-					$scope->addDynamicValue($notificationParameter->getKey(), $notificationParameter->getValue());
```

Comment:
Changed so that each template type will apply its own logic for when applying dynamic values to scope